new-cdc/eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/CdcPipelineConfiguration.java

```diff
@@ -19,14 +18,14 @@
 
 import javax.annotation.PostConstruct;
 import javax.annotation.PreDestroy;
-import java.io.IOException;
 import java.util.*;
 
 @Configuration
 public class CdcPipelineConfiguration {
```

Comment:
This is not an @Configuration class. It does not define any `@Beans`. It contains too much logic
All of the logic in this class should be moved into a CdcPipelineConfigurator class, which should be an `@Bean` defined by CdcPipelineConfiguration

Reply:
I will fix